explorer/client/src/components/longtext/Longtext.tsx

```diff
@@ -58,7 +60,7 @@ function Longtext({
         if (category === 'unknown') {
             textComponent = (
                 <span className={styles.longtext} onClick={navigateUnknown}>
-                    {text}
+                    {alttext ? alttext : text}
```

Comment:
The interface of this component is very odd because people usually  expect `text` to take prioritization over `alttext`, not the other way around. Could you make the interface clearer as a quick followup after we merge this?

Reply:
@666lcz The `alttext` is optional for this component while the `text` is required and the default. It's only used in cases where the display text is different from the link(text). I will make it clearer.